packages/server/src/api/routes/tests/search.spec.ts

```diff
@@ -59,7 +61,10 @@ describe.each([
   }
 
   async function createRows(rows: Record<string, any>[]) {
-    await Promise.all(rows.map(r => config.api.row.save(table._id!, r)))
+    // await config.api.row.bulkImport(table._id!, { rows })
```

Comment:
Delete if not needed

Reply:
Was the bulk import an issue? Is it a performance issue or what was the problem?